tools/src/test/resources/com/nedap/archie/flattener/siblingorder/openEHR-EHR-CLUSTER.siblingorderchild.v1.0.0.adls

```diff
@@ -22,9 +22,8 @@ description
 definition
     CLUSTER[id1.1] matches {    -- SiblingOrderChild
         /items matches {
-            -- CLUSTER [id5] (from parent)
-            CLUSTER[id6] occurrences matches {1}     -- Cluster 2
             CLUSTER[id7] occurrences matches {0}     -- Cluster 3
+            CLUSTER[id6] occurrences matches {1}     -- Cluster 2
```

Comment:
why this odd reordering?

Reply:
I think I moved something around for testing. It's back to normal ordering again.